entity-service/src/main/kotlin/com/egm/stellio/entity/authorization/AuthorizationService.kt

```diff
@@ -0,0 +1,27 @@
+package com.egm.stellio.entity.authorization
+
+interface AuthorizationService {
+
+    companion object {
+        const val USER_PREFIX: String = "urn:ngsi-ld:User:"
+        const val AUTHORIZATION_ONTOLOGY = "https://ontology.eglobalmark.com/authorization#"
+        const val R_CAN_READ = AUTHORIZATION_ONTOLOGY + "rCanRead"
+        const val R_CAN_WRITE = AUTHORIZATION_ONTOLOGY + "rCanWrite"
+        const val R_CAN_ADMIN = AUTHORIZATION_ONTOLOGY + "rCanAdmin"
+        val ADMIN_ROLES: Set<String> = setOf("admin")
+        val CREATION_ROLES: Set<String> = setOf("creator").plus(ADMIN_ROLES)
+        val READ_ROLES: Set<String> = setOf("reader").plus(CREATION_ROLES)
```

Comment:
it does not seem to be used ... related to the "discover" role we talked about?

Reply:
Yes indeed, this needs to be cleaned up.